### PR TITLE
Add rough rudimentary CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MV-GH**
Add rough rudimentary CONTRIBUTING.md

Please if you have any suggestions comment here

### CODE REVIEW

**Review by sabieber — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    * [Code quality](#code-quality)
+  * [Adding translations](#adding-translations)
+<!-- TOC -->
+
+<!-- prettier-ignore-end -->
+
+## Ways to contribute
+
+- Participate here and start answering questions.
+- File GitHub issues for big reports from /c/Jerboa. Watch the project for duplicate reports and
```
Comment (ZJouba) [inline CONTRIBUTING.md:20]:
```suggestion
- File GitHub issues for bug reports from /c/Jerboa. Watch the project for duplicate reports and
```

**Comment by ZJouba** [comment]
Someone mentioned Jetpack Compose on Matrix, maybe add a little something about that. Also, maybe a link to preferred Kotlin coding standards?

**Comment by MV-GH** [comment]
> Someone mentioned Jetpack Compose on Matrix, maybe add a little something about that. Also, maybe a link to preferred Kotlin coding standards?

Jetpack Compose is already mentioned in the README not sure what else to say about it

and I cant find the kotlin coding standards
https://github.com/jeremymailen/kotlinter-gradle their github pages is down for me

**Comment by ZJouba** [comment]
Could link to the ktlint rules https://pinterest.github.io/ktlint/0.49.1/rules/code-styles/?

**Comment by MV-GH** [comment]
Thx for finding that for me, I link to https://pinterest.github.io/ktlint/0.49.1/rules/standard/ instead since we just use the default rules.

**Review by ZJouba — APPROVED** [review]

**Comment by ZJouba** [comment]
Oh, maybe something about branch name standard as well?

**Comment by MV-GH** [comment]
There is a standard for branch names?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  * [Adding translations](#adding-translations)
+<!-- TOC -->
+
+<!-- prettier-ignore-end -->
+
+## Ways to contribute
+
+- Participate here and start answering questions.
+- File GitHub issues for bug reports from /c/Jerboa. Watch the project for duplicate reports and
+  link them to the first report so devs can easily close dupes.
```
Comment (dessalines) [inline CONTRIBUTING.md:21]: Don't do these weird manual line wraps, markdown renderers can handle long lines fine.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    * [Code quality](#code-quality)
+  * [Adding translations](#adding-translations)
+<!-- TOC -->
+
+<!-- prettier-ignore-end -->
+
+## Ways to contribute
+
+- Participate here and start answering questions.
+- File GitHub issues for bug reports from /c/Jerboa. Watch the project for duplicate reports and
```
Comment (dessalines) [inline CONTRIBUTING.md:20]: Add a link to https://lemmy.ml/c/jerboa

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  link them to the first report so devs can easily close dupes.
+- File new bug reports for issues you find.
+- Add missing translations
+- Code contributions
+
+## Application structure
+
+- Basic Modern Android Development (https://developer.android.com/series/mad-skills) tech stack (
```
Comment (dessalines) [inline CONTRIBUTING.md:28]: Use a markdown link.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  Compose, Navigation, Coroutines, AndroidX)
+- Guide to App Architecture (https://developer.android.com/topic/architecture), without domain
```
Comment (dessalines) [inline CONTRIBUTING.md:30]: Same

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+```shell
+./gradlew lintKotlin`
+```
+
+Or just run this to fix them
+
+```shell
+./gradlew formatKotlin
+```
```
Comment (dessalines) [inline CONTRIBUTING.md:56]: Thanks, these ones are going to be helpful.

**Review by dessalines — COMMENTED** [review]
Thanks for this, content all looks great. This doc will evolve over time anyway.

Just get rid of those hard line-wraps and I'll merge.

**Comment by dessalines** [comment]
Haha no standard for branch names or commit content, I'm not a stickler for that stuff. The code is what matters.

**Pull request merged**